modules/installing-rhv-example-install-config-yaml.adoc

```diff
@@ -70,8 +70,25 @@ platform:
 publish: External
 pullSecret: '{"auths": ...}'
 sshKey: ssh-ed12345 AAAA...
+compute:
+  - ...
+    sparse: false # add this line
+    format: raw   # add this line
+controlPlane:
+  platform:
+    ovirt:
+      ..
+      sparse: false # add this line <1>
+      format: raw   # add this line <2>
 ----
 
+<1> Sparse (default: `true`) - Setting this option to `false` enables preallocation of disks. Setting `sparse` to `true` with `format` set to `raw` is not available for file storage domains (such as NFS).
```

Comment:
"Setting sparse to true with format set to raw is not available for file storage domains (such as NFS)."

This is incorrect, and after double checking with @janosdebugs we've found that it was also incorrect in the New feature enablement presentation for 4.11.

This combination is available on file storage, but is not available on block storage.

So it should say RAW + sparse=true is not supported on block storage.